test/e2e/test-deployment.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: DEPLOYMENT_NAME
```

Comment:
could use golangs text/templates here instead. 

Not clear how htis one diffeers from nginx one.